src/main/kotlin/no/nav/syfo/Environment.kt

```diff
@@ -179,6 +180,7 @@ data class DbEnv(
 
 data class ToggleEnv(
     val sendAktivitetspliktForhandsvarsel: Boolean,
+    val sendArbeidsuforhetForhandsvarsel: Boolean,
```

Comment:
Trenger dere toggles? Ikke noe problem for vår del altså :D Men kanskje enklere å toggle på deres side hvorvidt det skal sendes til esyfovarsel eller ikke, eller evnt bare deploye featurebranch til dev? Om dere likevel vil ha det her så er det no stress, bare huske å fjerne etterpå når det ikke trengs lenger

Reply:
Tenker vi like greit kan toggle det på vår side :+1: